player.gd

```diff
@@ -5,8 +5,11 @@ signal hit
 var speed = 0
 var swim_impulse = 150
 var coasting_deceleration = 150
+var air_gravity = 400
```

Comment:
I know the tutorial used their own physics like this, but have you looked into the physics engine at all?  I would just be able to add some properties to the Air node to override gravity instead of this manual flagging.  (It works just fine, just wondering)

Reply:
I would be interested in seeing what we can do with the physics engine!